webrtc-stats.html

```diff
@@ -1234,6 +1235,17 @@ <h2>
                   Opus).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>fecPacketsDiscarded</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP FEC packets received for this SSRC where the error correction
+                  payload was discarded due to having received the original packet. This is a
```

Comment:
Discard can also happen FEC packets arriving after the recovery window has expired.

Reply:
Total number of RTP FEC packets received for this SSRC where the error correction payload was discarded by the application. This may happen 1. if all the source packets protected by the FEC packet were received, or 2. if the FEC packet arrived late, i.e., outside the recovery window, and the lost RTP packets have already been skipped during playout.